assets/blocks/restricted-content/index.js

```diff
@@ -0,0 +1,75 @@
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/components';
+import edit from './edit';
+import save from './save';
+import metadata from './block';
+import { createBlock } from '@wordpress/blocks';
+
+export default {
+	title: __( 'Restricted Course Content', 'sensei-lms' ),
+	description: __(
+		'Content inside this container block can be restricted to a specified group of users. Possible options are enrolled users, not enrolled users and users which completed the course.',
```

Comment:
Let's change it to "users who have completed the course".

Reply:
Fixed in 34a03073f7a505361d92fe2c8885f65abd86d3cc